api/server/server.go

```diff
@@ -624,6 +624,17 @@ func (s *Server) getContainersLogs(version version.Version, w http.ResponseWrite
 		closeNotifier = notifier.CloseNotify()
 	}
 
+	c, err := s.daemon.Get(vars["name"])
```

Comment:
Yes but sometimes this is needed and we can't do anything (like hijacking the request and not being able to return 404, same case as this one I suppose)